share/spack/gitlab/cloud_pipelines/stacks/ml-linux-ppc64le-cpu/spack.yaml

```diff
@@ -0,0 +1,99 @@
+spack:
+  view: false
+  packages:
+    all:
+      require: target=ppc64le
+      variants: ~cuda~rocm
+
+  specs:
+    # Horovod
+    # See pytorch failures
+    #- py-horovod tensor_ops=mpi controllers=mpi frameworks=pytorch
```

Comment:
Looks like, for some reason, py-horovod still depends on py-tensorboard, even if it's using the pytorch framework and not tensorflow ( https://gitlab.spack.io/spack/spack/-/pipelines/480840 ).

Reply:
For some reason it's building py-pytorch-lightning 1.5.3 instead of the latest version. If you can figure that out it will no longer depend on tensorboard. But would be great to get tensorboard working.